test/data/case5_dc.m

```diff
@@ -0,0 +1,65 @@
+% NESTA v0.6.0
```

Comment:
You might want to make a case, specifically for testing the dc line model.  For example one where flow on the dc line is quite high and significant losses are incurred.  In the unit test, you might also want to check the amount of loss that occurs across the line.